src/components/client/core/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,43 @@
+import { cloneElement } from "react";
+
+import { Text } from "components/universal";
+import { panda } from "generated/panda/jsx";
+import { breadcrumb } from "generated/panda/recipes";
+
+import type { ReactElement, ReactNode } from "react";
+
+export interface Props {
+  pathname: string;
+  baseUrl?: string;
+  seperator: ReactNode;
+}
+
+/**
+ * Core UI breadcrumb.
+ */
+const Breadcrumb = ({ pathname, baseUrl = "Home", seperator }: Props) => {
+  if (pathname.startsWith("https://")) {
+    throw new Error("The pathname prop should not include 'https://'.");
+  }
+
+  const breadcrumbItems: (string | ReactNode)[] = [baseUrl];
```

Comment:
I think this type annotation can be simplified to just `ReactNode[]`, since a `string` is a type subset of a `ReactNode`